spec/default_spec.rb

```diff
@@ -209,7 +209,7 @@
       let(:template) { chef_run.template('/opt/local/etc/rsyslog.d/50-default.conf') }
 
       it 'creates the template' do
-        expect(chef_run).to render_file(template.path).with_content('Default rules for rsyslog.')
+        expect(chef_run).to render_file(template.path).with_content('Generated by Chef')
```

Comment:
```suggestion
        expect(chef_run).to render_file(template.path).with_content('Generated by Chef Infra Client')
```

Reply:
This one should be dealt with in the latest commit - and I fixed the long-failing tests.